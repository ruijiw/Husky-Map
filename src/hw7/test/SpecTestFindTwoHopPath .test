# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for one character have two hop("edge") path to another character.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary CATLYN VI

FindPath dictionary RIVEN SIVIR