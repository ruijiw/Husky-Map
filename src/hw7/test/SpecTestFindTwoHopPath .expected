# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for one character have two hop("edge") path to another character.

loaded graph dictionary

path from CATLYN to VI:
CATLYN to TEEMO with weight 1.000
TEEMO to VI with weight 0.500
total cost: 1.500

path from RIVEN to SIVIR:
RIVEN to TEEMO with weight 1.000
TEEMO to SIVIR with weight 0.500
total cost: 1.500